satpy/enhancements/viirs.py

```diff
@@ -0,0 +1,20 @@
+from trollimage.colormap import Colormap
+import numpy as np
+from satpy.enhancements import apply_enhancement
+
+
+def water_detection(img, **kwargs):
```

Comment:
Could you add some docstrings to this function describing what it tries to do. First line should be a single sentence and fit on one line, then a blank line, then whatever you want.